client.go

```diff
@@ -22,6 +22,7 @@ func readTeamApi(url string, login string, passwd string) []Team {
 	if err != nil {
 		log.Print(err)
 		os.Exit(1)
+		// Comment
```

Comment:
Could you explain a bit the different about using `Exit` in here instead of `panic`?

Reply:
panic allows you to recover from the problem. It sets an error, execute deferred function... 
In my small example I directly call Exit which immediately quit the app.